src/test/java/teammates/test/cases/browsertests/InstructorCourseDetailsPageUiTest.java

```diff
@@ -272,24 +271,18 @@ private InstructorCourseDetailsPage getCourseDetailsPage() {
     }
 
     private boolean hasStudentReceivedReminder(String courseName, String courseId, String studentEmail)
-            throws MaximumRetriesExceededException {
+            throws Throwable {
```

Comment:
Use `Exception`? TEAMMATES is kinda stuck with it because of some not so good design of mixing exceptions with errors. Some errors in Java are pretty much unrecoverable and we can't do anything useful with it such as `OutOfMemmoryError`